index.html

```diff
@@ -1005,7 +978,7 @@ <h1>DID URL Dereferencing</h1>
 			this MUST NOT be empty. Properties defined by this specification are in <a
 				href="#did-url-dereferencing-metadata"></a>. If the dereferencing is not
 			successful, this structure MUST contain an <code>error</code> property
-			describing the error.
+			describing the error. See Section <a href="#errors"></a>.
```

Comment:
Propose to add this same sentence also in the `didResolutionMetadata` section here:
https://github.com/w3c/did-resolution/pull/146/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R463

Reply:
Hmm, strange I cant seem to accept this suggestion